### PR TITLE
ENH Remove unused Translatable code

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-framework/issues/10548

Failing unit test is unrelated

### CODE REVIEW

File: code/Search/SearchForm.php [hunk 1]
```diff
@@ -133,18 +121,6 @@ public function getResults()
         // set language (if present)
         $locale = null;
         $origLocale = null;
```
Comment (michalkleiner) [inline code/Search/SearchForm.php]:
```suggestion
```

Comment (emteknetnz) [reply code/Search/SearchForm.php]: Updated

Failing unit test is unrelated

File: code/Controllers/ContentController.php [hunk 2]
```diff
@@ -415,9 +386,7 @@ public function SiteConfig()
 
     /**
      * Returns an RFC1766 compliant locale string, e.g. 'fr-CA'.
-     * Inspects the associated {@link dataRecord} for a {@link SiteTree->Locale} value if present,
-     * and falls back to {@link Translatable::get_current_locale()} or {@link i18n::default_locale()},
-     * depending if Translatable is enabled.
+     * Inspects the associated {@link dataRecord} for a {@link SiteTree->Locale} value if present, and falls back to {@link i18n::default_locale()}
```
Comment (ChloeSartorelli) [inline code/Controllers/ContentController.php]: You've changed this to no longer look for a `SiteTree->Locale` value. We should either look for that value (if anything in core would add it) or remove the reference to that in the phpdoc

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: code/Controllers/ContentController.php (hunk 2 above)
Comment (emteknetnz) [reply code/Controllers/ContentController.php]: Have removed the phpdoc

**Review by GuySartorelli — APPROVED** [review]
Failed unit tests do indeed seem unrelated. Changes look good to me.

**Pull request merged**